scripts/create-instance-and-test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -eu
+source ./scripts/rundeck.sh
+
+JOB_CREATE_TEST_INSTANCE="7170b8bc-10d2-4099-b424-8e194a5d5884"
+
+INSTANCE_ENV='stg'
+INSTANCE_LOCALE='en'
+#We need a different name for photos and drive build otherwise we'll try to create the same instance twice in travis.
+INSTANCE_ID="${COZY_APP_SLUG,,}"$TRAVIS_PULL_REQUEST
+
+
+echo "↳ ℹ️  Creating instance..."
+INSTANCE_CREATE_EXECUTION_ID=$(runRundeckJob $JOB_CREATE_TEST_INSTANCE "-commitid $INSTANCE_ID -environment $INSTANCE_ENV -locale $INSTANCE_LOCALE")
+getRundeckStatus $INSTANCE_CREATE_EXECUTION_ID
+INSTANCE_CREATE_STATUS=$?
+echo $INSTANCE_CREATE_STATUS
+
+if [ $INSTANCE_CREATE_STATUS == 0 ] ; then
```

Comment:
If the creation failed, will the execution stop and an error be reported?

Reply:
if the execution status return `fail` -> error & message
if the execution status body contains `error` -> error & message (except for the `deploy job`which always return the log with `level=error`)

I just add a check on whether the POST return something or not which display the request if nothing is return